test/mbr_update_test.rb

```diff
@@ -28,6 +31,16 @@
       )
     end
 
+    context "on s390" do
+      before do
+        allow(Yast::Arch).to receive(:architecture).and_return("s390_64")
+      end
+
+      it "does nothing except returning true" do
+        expect(subject.run).to eq true
```

Comment:
and also add it to when partition activated? this mean, that for any new feature I need to verify that it really do nothing, that is a bit overkill for me.